frontend/web/components/EditPermissions.tsx

```diff
@@ -241,6 +241,8 @@ const _EditPermissionsModal: FC<EditPermissionModalType> = forwardRef(
           skip:
             !id ||
             envId ||
+            // TODO: In the future, perhaps use inherited components to avoid issues with merging multiple contexts into a single component.
```

Comment:
We discussed creating an issue for this and linking to it here, right?

Reply:
Yes, created and added